src/tools/blast/blastcommandline.jl

```diff
@@ -0,0 +1,24 @@
+# wrapper for BLAST+ command line functions
+
+using LightXML
+
+type BlastResult
+    queryname::String
+    subjectname::String
+    query::NucleotideSequence
+    subject::NucleotideSequence
+end
+    
+
+
+"""
+Runs blastn on `query` against `subject`. May include optional `flags` such as
```

Comment:
Do you mean something like:

    # Basic Functions
    # ---------------

?